### ISSUE TITLE
Start

### ISSUE THREAD

**Issue by rogeralsing** [ActorModelDotNet/actormodeldotnet.github.com#12]
@sergeybykov I've opened up a few issues around topics that I think could be interesting to compare, just sort of mind mapping to begin with.
Feel free to add more or alter the existing ones.

Once we have a few topics that we agree on, we could give it a shot to describe and compare how the different frameworks handle/behave in each topic.

Sounds fair?

**Comment by sergeybykov** [comment]
Yes, looks like a good start. We may also need a high-level architectural topic for key tenets and tradeoffs that would help understand some of the choices made on particular issue/design alternatives from the big picture perspective.

Is this open to everyone for comments?

**Comment by rogeralsing** [comment]
Yes, everyone is welcome to contrib/discuss the topics

**Comment by sergeybykov** [comment]
Great. I added https://github.com/ActorModelDotNet/actormodeldotnet.github.com/issues/14 for high-level architectural goals and principles.
